lib/onboarding.dart

```diff
@@ -0,0 +1,117 @@
+import 'package:flutter/material.dart';
+import 'package:introduction_screen/introduction_screen.dart';
+
+class onboarding extends StatefulWidget {
+  @override
+  _onboardingState createState() => _onboardingState();
+}
+
+class _onboardingState extends State<onboarding> {
+  List<PageViewModel> getPages() {
+    return [
+      //-----------------------------------------------------------Welcome
+      PageViewModel(
+          image: Image.asset("assets/images/logo.png"),
+          title: "Welcome to Eureka!",
+          body:
+              "Eureka is a platform for peers to help each other by answering questions \n\n Everyone can help and even get paid for help!",
+          footer: Text(""),
+          decoration: const PageDecoration(
+              //pageColor: Color(0xfffdfafc),
+              )),
+      //-----------------------------------------------------------Incentives
+      PageViewModel(
+          image: Image.asset("assets/images/money.gif"),
+          title: "Incentives",
+          body:
+              "Eureka allows posting questions as on any regular forum, BUT also gives you an option of posting questions with an incentive \n\n"
+              " Need answer ASAP? \n\n Try adding incentive to your question \n Paid questions get answers a lot faster!",
+          footer: Text(""),
+          decoration: const PageDecoration(
+              //pageColor: Color(0xffB9AEFB),
+              )),
+      //-----------------------------------------------------------Collaboration
+      PageViewModel(
+        image: Image.asset("assets/images/onboarding2.gif"),
+        title: "Rich collaboration platform",
+        body:
+            "Eureka provides you with every tool to conveniently collaborate on any problem ",
+        decoration: const PageDecoration(
+            //pageColor: Color(0xfffdfafc),
+            ),
+        footer: Text("Choose from Text chat, Video chat, or Live Video Call"),
+      ),
+      //-----------------------------------------------------------Privacy
+      PageViewModel(
+        image: Image.asset("assets/images/privacy.png"),
+        title: "Your Privacy Matters!",
+        body:
+            "All communication happens in the app, so you don't have to worry about giving out your personal information to strangers",
+        decoration: const PageDecoration(
+            //pageColor: Color(0xfffdfafc),
+            ),
+        footer: Text(""),
+      ),
+      //-----------------------------------------------------------Categories
+      PageViewModel(
+        image: Image.asset("assets/images/onboarding1.gif"),
+        title: "Multiple categories",
+        body:
+            "Eureka has several most popular categories for questions and answers",
+        decoration: const PageDecoration(
+          pageColor: Color(0xfffdfafc),
+        ),
+        footer: Text("Lifestyle  Education  Electronics  Household"),

```

Comment:
Here too the footer font size is a little small.
![image](https://user-images.githubusercontent.com/38572314/105776376-5504cb80-5f1d-11eb-9835-f001726c6b3a.png)

Reply:
sure, no problem